frontend/fingertips-frontend/components/molecules/Pill/index.tsx

```diff
@@ -38,12 +38,18 @@ const RemoveAreaButton = styled('button')({
   padding: '5px',
 });
 
-interface PillProps {
+export interface PillProps {
   children: React.ReactNode;
   selectedFilterId?: string;
   removeFilter?: (filterId: string) => void;
   isFullWidth?: boolean;
+
+  // used to generate an aria-label
   ariaLabelPostfix?: string;
+
+  // If true the pill should only allow the data to be viewed, and have no
+  // behaviour that can cause the UI to be changed.
+  isViewOnly?: boolean;
```

Comment:
the presence of the `removeFilter` prop does the same thing. Do we need this additional prop? Just don't provide the removeFilter function when we don't need the `X`